File: Java/ReverseOfGivenNumber.java
//Write a java program to reverse given number

import java.util.Scanner;

public class ReverseOfGivenNumber 
{
    public static void main(String[] args) 
    {
        int n;
        System.out.println("Enter the number : ");
        Scanner scanner = new Scanner(System.in);
        n = scanner.nextInt();

        int rem, result = 0;

        while (n != 0) 
        {
            rem = n % 10;
            result = result * 10 + rem;
            n = n / 10;
        }
        System.out.println("The reverse of given number is " + result);
    }
}

File: C#/ReverseOfGivenNumber.cs
//Write a c# program to reverse given number

using System;

public class ReverseOfGivenNumber
{
    static void Main()
    {
        int n;
        Console.WriteLine("Enter the number : ");
        n = int.Parse(Console.ReadLine());

        int rem, result = 0;

        while(n != 0)
        {
            rem = n % 10;
            result = result*10 + rem;
            n = n / 10;
        }
        Console.WriteLine("The sum of digits is " + result);
    }
}